README.es.md

```diff
@@ -0,0 +1,173 @@
+![Drupal VM Logo](https://raw.githubusercontent.com/geerlingguy/drupal-vm/master/docs/images/drupal-vm-logo.png)
+
+[![Build Status](https://travis-ci.org/geerlingguy/drupal-vm.svg?branch=master)](https://travis-ci.org/geerlingguy/drupal-vm) [![Documentation Status](https://readthedocs.org/projects/drupal-vm/badge/?version=latest)](http://docs.drupalvm.com) [![Packagist](https://img.shields.io/packagist/v/geerlingguy/drupal-vm.svg)](https://packagist.org/packages/geerlingguy/drupal-vm) [![Docker Automated build](https://img.shields.io/docker/automated/geerlingguy/drupal-vm.svg?maxAge=2592000)](https://hub.docker.com/r/geerlingguy/drupal-vm/) [![](https://images.microbadger.com/badges/image/geerlingguy/drupal-vm.svg)](https://microbadger.com/images/geerlingguy/drupal-vm "Get your own image badge on microbadger.com") [![irc://irc.freenode.net/drupal-vm](https://img.shields.io/badge/irc.freenode.net-%23drupal--vm-brightgreen.svg)](https://riot.im/app/#/room/#drupal-vm:matrix.org)
+
+[Drupal VM](https://www.drupalvm.com/) Es un VM para Drupal, construido con Ansible.
+
+Drupal VM hace que la construcción de entornos de desarrollo Drupal sea rápido y fácil, y presenta a los programadores al maravilloso mundo del programador Drupal en máquinas virtuales o contenedores Docker (en vez del viejo programador MAMP/WAMP-based).
+
+Instalará los siguientes en un Ubuntu 16.04 (predeterminado) linux VM:
+
+  - Apache 2.4.x (or Nginx)
```

Comment:
Can you also translate the contents of this list (e.g. `or` to `o`, `for accessing databases directly`, etc.)?